dlt/common/runners/stdout.py

```diff
@@ -24,6 +25,45 @@ def exec_to_stdout(f: AnyFun) -> Iterator[Any]:
             print(encode_obj(rv), flush=True)
 
 
+def iter_std(venv: Venv, command: str, *script_args: Any) -> Iterator[Tuple[int, str]]:
+    """Starts a process `command` with `script_args` in environment `venv` and returns iterator
+    of (filno, line) tuples where `fileno` is 1 for stdout and 2 for stderr. `line` is
```

Comment:
Maybe we should create a simple enum to denote 1 for stdout and 2 as sdterr.

Reply:
I will create a Literal with those codes, they correspond to unix file numbers for std so IMO they are fine